zanata-model/pom.xml

```diff
@@ -320,8 +320,10 @@
     </dependency>
 
     <dependency>
-      <groupId>xom</groupId>
+      <!-- New group Id -->
+      <groupId>com.io7m.xom</groupId>
       <artifactId>xom</artifactId>
+      <version>1.2.10</version>
```

Comment:
If you have the version in server/pom.xml's dependencyManagement, you don't need to repeat it here.